incubator/astro/values.yaml

```diff
@@ -0,0 +1,307 @@
+image:
+  repository: quay.io/fairwinds/astro
+  tag: "v1.2.0"
+  pullPolicy: IfNotPresent
+resources:
+  limits:
+    cpu: 100m
+    memory: 128Mi
+  requests:
+    cpu: 100m
+    memory: 128Mi
+nodeSelector: {}
+tolerations: []
+affinity: {}
+datadog:
+  apiKey: ""
+  appKey: ""
+definitionsPath: conf.yml
+owner: astro
+DryRun: false
+custom_config:
+  enabled: false
```

Comment:
For CI e2e testing purposes, can you add a ci/test-values.yaml that enables this feature?  That way we can test that it works.

Reply:
sure, test-values.yaml added.